std/os/uefi/tables/boot_services.zig

```diff
@@ -0,0 +1,66 @@
+const TableHeader = @import("table_header.zig").TableHeader;
+
+// TODO this doesn't work yet?!
+var boot_services_exited: bool = false;
+
+pub const BootServices = struct {
+    hdr: TableHeader,
+    _raise_tpl: usize, // TODO
```

Comment:
why the `_` prefix?

Reply:
These attributes aren't meant to be directly used as there's no guarantee they actually exist. The wrapper methods are there to check if they do.